src/server/hll_family_test.cc

```diff
@@ -26,6 +29,20 @@ TEST_F(HllFamilyTest, Simple) {
   EXPECT_EQ(CheckedInt({"pfcount", "key"}), 1);
 }
 
+TEST_F(HllFamilyTest, Promote) {
+  int unique_values = 2000;
+  for (int i = 0; i < unique_values; ++i) {
+    std::string newkey = GenerateUniqueValue(i);
+    // promote was triggered at 1660th insertion. But how to verify that?
+    Run({"pfadd", "key", newkey});
+  }
+  // HyperLogLog is an advanced data structure in Redis, primarily used for cardinality estimation
+  // (counting unique elements) of massive datasets (up to 2^64 elements). Its characteristics
+  // include high speed and low memory footprint (12KB). However, its computations come with a
+  // margin of error, with a standard error rate of 0.81%. got 0.83% error rate when unique_values =
+  // 200000 EXPECT_LT(std::abs(Run({"pfcount", "key"}) - unique_values)/unique_values, 0.0081);
```

Comment:
```suggestion
  // 200000
  EXPECT_LT(std::abs(Run({"pfcount", "key"}) - unique_values)/unique_values, 0.1);
```

This should be high enough so that it wouldn't fail in practice, but will have some test value, no?